etcdmain/config.go

```diff
@@ -26,12 +26,13 @@ import (
 	"runtime"
 	"strings"
 
+	"sigs.k8s.io/yaml"
+
 	"go.etcd.io/etcd/embed"
 	"go.etcd.io/etcd/pkg/flags"
 	"go.etcd.io/etcd/pkg/types"
 	"go.etcd.io/etcd/version"
 
-	"github.com/ghodss/yaml"
```

Comment:
replace this line with sigs.k8s.io/yaml instead of creating a new import block at line 29

Reply:
placed this at the bottom instead of simply replacing the existing ghodss dependency as the 3rd party imports should be sorted alphabetically.